client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -69,33 +68,42 @@ $deisgn-selector-selection-space: 175px;
 	@include design-selector-show();
 }
 
-.page-layout-selector__container {
+.design-selector__page-layout-backdrop {
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
+}
+
+.design-selector__page-layout-container {
 	position: absolute;
 	top: $design-selector-header-vertical-shift + $deisgn-selector-selection-space;
+	width: 100%;
+	background: $white;
+	box-shadow: gray 0 0 12px; // use a sass variable
```

Comment:
Yeah, indeed something from here;

https://github.com/WordPress/gutenberg/blob/8084e23a8c5364e139790c208a9d9d593b0887b9/packages/base-styles/_colors.scss

Would `$dark-gray-300: #6c7781` work? cc @dwolfe  

Not a blocker for merging, we can update later.

Reply:
BTW denoting these with `@TODO` is useful as some IDEs highlight them or list them conveniently in one place.

Reply:
`$dark-gray-300` looks like a good match, let's use that for now.